spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/Options.scala

```diff
@@ -39,7 +39,8 @@ object Options {
     warmupEpoch: Option[Int] = None,
     gradientL2NormThreshold: Option[Double] = None,
     gradientMin: Option[Double] = None,
-    gradientMax: Option[Double] = None
+    gradientMax: Option[Double] = None,
+    visualization: Boolean = true
```

Comment:
Also please update the ReadMe.md file in the same folder to let user know how to use this new option.